test/ir/inference/auto_scan_test.py

```diff
@@ -747,10 +746,13 @@ def random_to_skip():
             if not skip_baseline:
                 # baseline: gpu run, we only test float32
                 gpu_config = self.create_inference_config(use_trt=False)
+                prog_config = prog_config.set_input_type(
```

Comment:
We reduce the input of the baseline to a smaller number of bits, which could potentially narrow the gap between the fp16 and fp32 computation.